.circleci/config.yml

```diff
@@ -171,15 +171,32 @@ jobs:
             MBT_VERSION=$(cat ./VERSION)
             echo "Image release: ${MBT_VERSION}"
             git checkout docker
+            #Push to Docker Hub
+            docker login -u ${DOCKER_HUB_USER} -p ${DOCKER_HUN_TOKEN}
```

Comment:
Is ${DOCKER_HUN_TOKEN} correct, looks like a typo to me!? I'd expect DOCKER_HUB_TOKEN with a 'B' here.